src/ripple/protocol/SField.h

```diff
@@ -387,6 +387,7 @@ extern SF_U64 const sfBaseFee;
 extern SF_U64 const sfExchangeRate;
 extern SF_U64 const sfLowNode;
 extern SF_U64 const sfHighNode;
+extern SF_U64 const sfAltOwnerNode;
```

Comment:
Consider `sfDestinationNode`, as suggested in the Checks proposal: https://ripplelabs.atlassian.net/wiki/spaces/RIPD/pages/111566093/Checks.  I think `sfDestinationNode` is descriptive of the intent for both Checks and Escrow.  It would probably work well for Payment Channels as well.

Reply:
:+1: done!